Rakefile

```diff
@@ -5,6 +5,13 @@ namespace :db do
     task :create do
       sh "createdb travis_#{env}" rescue nil
       sh "psql -q travis_#{env} < #{Gem.loaded_specs['travis-migrations'].full_gem_path}/db/structure.sql"
+
+      # logs database
+      require 'sequel'
+      Sequel.extension(:migration)
+      db = Sequel.connect(:adapter => 'postgres', :database => "travis_#{env}")
```

Comment:
Good point. This should be using `config.logs_database`, since that then also supports the `LOGS_DATABASE_URL` env var.